de.dlr.sc.virsat.model.edit/src/de/dlr/sc/virsat/model/concept/types/structural/ABeanStructuralElementInstance.java

```diff
@@ -109,6 +131,34 @@ public Command remove(EditingDomain ed, IBeanCategoryAssignment bca) {
 		return bcaHelper.getAllBeanCategories(sei, catBeanClazz);
 	}
 	
+	@Override
+	public List<BeanCategoryAssignment> getCategoryAssignments() {

```

Comment:
There seems to be some inconsistency between using the abstract classes and the generic classes.
E.g. the getChildren uses the abstract class for seis and the getCategoryAssignments uses the generic class.

Reply:
I added comments, why we do this

Reply:
I don't see the comments.